src/sagemaker/feature_store/feature_utils.py

```diff
@@ -0,0 +1,334 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Utilities for working with FeatureGroups and FeatureStores."""
+from __future__ import absolute_import
+
+import re
+import logging
+
+from typing import Union
+from pathlib import Path
+
+import pandas
+import boto3
+from pandas import DataFrame, Series, read_csv
+
+from sagemaker.feature_store.feature_group import FeatureGroup
+from sagemaker.session import Session
+
+logger = logging.getLogger(__name__)
+
+
+def get_session_from_role(region: str, assume_role: str = None) -> Session:
+    """Method use to get the :class:`sagemaker.session.Session`  from a role and a region.
+
+    Description:
+        Helpful in case it's invoke from a session with a role without permission it can assume
+        another role temporarily to perform certain tasks.
+
+    Args:
+        assume_role: role name
+        region: region name
+
+    Returns:
+    """
+    boto_session = boto3.Session(region_name=region)
+
+    # It will try to assume the role specified
+    if assume_role:
+        sts = boto_session.client(
+            "sts", region_name=region, endpoint_url="https://sts.eu-west-1.amazonaws.com"
+        )
+
+        metadata = sts.assume_role(RoleArn=assume_role, RoleSessionName="SagemakerExecution")
+
+        access_key_id = metadata["Credentials"]["AccessKeyId"]
```

Comment:
use `.get` method here to prevent exception if Credentials is not present in metadata.

Reply:
done